### PR TITLE
:lipstick: :mortar_board: Intro topic --- Small cleanup :microscope: 

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
Small cleanup 
- Make the double quotes look different
- Remove period 

### Why
Cleaner and more consistent

### CODE REVIEW

File: src/site/topics/intro/introduction.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ The main focus of this course is:
 
 
 Data Structures
-===============
+===============Unit Testing Collections Topic --- Make a new topic for testing collections
```
Comment (twentylemon) [inline src/site/topics/intro/introduction.rst:23]: wut

Comment (jameshughes89) [reply src/site/topics/intro/introduction.rst:23]: wut indeed

Comment (jameshughes89) [reply src/site/topics/intro/introduction.rst]: fixed

File: src/site/topics/intro/introduction.rst [hunk 2]
```diff
@@ -223,7 +223,7 @@ Can we Write Code Now?
 
         * We don't need to make an instance of the class to use the method
 
-    * A good example of this in Java is the `Math class <https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html>`_.
+    * A good example of this in Java is the `Math class <https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html>`_
```
Comment (twentylemon) [inline src/site/topics/intro/introduction.rst:226]: maybe better to link to jdk17  https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Math.html

**Review by twentylemon — APPROVED** [review]

File: src/site/topics/intro/introduction.rst (hunk 2 above)
Comment (jameshughes89) [reply src/site/topics/intro/introduction.rst:226]: good catch

**Pull request merged**